package/functions/funcs/variablesCount.js

```diff
@@ -1,7 +1,47 @@
 module.exports = async (d) => {
+
+const inside = d.unpack()
+
+	const err = d.inside(inside)
+
+		if (err) return d.error(err)
+
+    
+
+let res;
+
+    let [type] = inside.splits
+
+if(type == "data"){
+
+res = await d.client.db.all("main") 
+
+res = res.length
+
+}
+
+else if(type =="vars"){
+
+res = Object.keys(d.client.variables).length
+
+}
+
+else {
+
+d.error(`Invalid Type Provided In $variablesCount${inside}`)
+
+}
```

Comment:
Minimizing it would be a better idea rather than doing a long code.

```js
const [type = "null"] = inside.splits || new Array();

if (!["data", "vars"].includes(type)) return d.error(`:x: Invalid Type Provided In $variablesCount${inside.total ? inside.total : ""}`)

const res = ( type === "data" ? ( await d.client.db.all("main") ).length : Object.keys(d.client.variables).length )

Reply:
oo ok

Reply:
@Kino7916 I minimised the code it should be better now